app/assets/javascripts/ng-control/operations.js

```diff
@@ -1,3 +1,5 @@
+var no_race = "init";
```

Comment:
Ben, this is a flag used to prevent a race-condition when clicking on the [ Activity Reports ] tab then another tab.  That is a CURRENT problem in the CURRENT production code - this is my hack to fix it.